.clang-format

```diff
@@ -1,7 +1,15 @@
+---
+Language: Cpp
 BasedOnStyle: LLVM
 PointerAlignment: Left
 IndentCaseLabels: true
 ColumnLimit: 100
 ContinuationIndentWidth: 2
 ConstructorInitializerIndentWidth: 2
-AlignAfterOpenBracket: DontAlign
+SpaceAfterTemplateKeyword: false
+BinPackArguments: false
+BinPackParameters: false
+---
+Language: JavaScript
+DisableFormat: true
```

Comment:
Maybe we don't want to disable formatting for JS here.  It can be useful for to format JS code I think.  How about we do just `BasedOnStyle: LLVM` + `ColumnLimit: 100` here?

Reply:
Do you mean column limit 100 for C++ and JS? I at least would like to make it 80 for C++...

Reply:
Sure, I'll add BasedOnStyle: LLVM + ColumnLimit: 100 here